source/moto_payments/index.html.md.erb

```diff
@@ -16,9 +16,11 @@ This means you can take payments:
 
 These payments are also known as Mail Order / Telephone Order (MOTO) payments.
 
-You cannot take MOTO payments if your payment service provider (PSP) is SmartPay.
+You cannot take MOTO payments:
 
-You also cannot take MOTO payments through a [payment link](https://www.payments.service.gov.uk/govuk-payment-pages/) - [contact us](/support_contact_and_more_information/#contact-us) if you'd like us to develop this feature.
+- if your payment service provider (PSP) is SmartPay
+- through a [payment link](https://www.payments.service.gov.uk/govuk-payment-pages/) - [contact us](/support_contact_and_more_information/#contact-us) if you'd like us to develop this feature
+- if you are not PCI compliant
```

Comment:
if you are not PCI compliant for phone payments?
they really shouldn't be taking any payment if they are not PCI compliant, but we handle most of it for online payments. they have more responsibilities for compliance for phone payments.

Reply:
might be one to check with mark willcox/David